application/src/main/java/org/opentripplanner/transit/api/request/TripOnServiceDateRequest.java

```diff
@@ -5,11 +5,12 @@
 import org.opentripplanner.framework.collection.ListUtils;
 import org.opentripplanner.transit.model.framework.FeedScopedId;
 import org.opentripplanner.transit.model.timetable.TripAlteration;
+import org.opentripplanner.transit.model.timetable.TripOnServiceDate;
 
-/*
+/**
  * A request for trips on a specific service date.
  *
- * This request is used to retrieve TripsOnServiceDates that match the provided criteria.
+ * This request is used to retrieve {@link TripOnServiceDate}s that match the provided criteria.
  * At least one operatingDay must be provided.
  */
 public class TripOnServiceDateRequest {
```

Comment:
We should change `lines` to `routes` below as well.

Reply:
I'm also unclear about the use of 'agency' vs 'authority' - is this another case of OTP vs Netex terminology?

Reply:
Yes, in most cases OTP 'agency' is NeTEx 'authority' - but Netex has a richer model with operator and bookinginfo witch in some cases overlap with agency. There are also deployment variations - operator/authority does not mean the same thing in Norway and Netherlands (I do not know the details here).

Reply:
Done.
<!-- Reviewable comment -OBPIDEK7psCFfXA8ev2:b-896fix -->
<!-- Sent from Reviewable.io -->